packages/api/package.json

```
@@ -13,8 +13,8 @@
     "lint:types": "tsc --noEmit",
     "prebuild": "rm -rf dist/; npm run version",
     "prepack": "npm run build",
-    "test": "jest --coverage $(find test -name '*.test.ts' -not -path '*/smoketest.test.ts')",
-    "test:smoke": "npx jest test/cli/codegen/languages/typescript/smoketest.test.ts",
+    "test": "vitest run --coverage",
+    "test:smoke": "vitest run --config=vitest-smoketest.config.ts ",
```

similar to my suggestion in this comment

```suggestion
    "test": "vitest --coverage",
    "test:smoke": "vitest --config=vitest-smoketest.config.ts ",
```